_includes/openseeds-header.html

```diff
@@ -0,0 +1,78 @@
+        <div class="navbar-end">
+          <div class="navbar-item has-dropdown is-hoverable">
+            <div class="navbar-dropdown">
+              <a class="navbar-item" href="{% link openseeds/index.md %}#applications"> Applications </a>
+              <a class="navbar-item" href="{% link about.md %}"> The program itself </a>
+              <a class="navbar-item" href="{% link about.md %}#projects"> Projects </a>
+              <a class="navbar-item navbar-previous-title" href="{% link about.md %}#the-roles"> Role description </a>
+              <a class="navbar-item" href="{% link about.md %}#mentees"> Mentees </a>
+              <a class="navbar-item" href="{% link about.md %}#mentors"> Mentors </a>
+              <a class="navbar-item" href="{% link about.md %}#experts"> Experts </a>
+              <a class="navbar-item" href="{% link about.md %}#facilitators"> Facilitators </a>
+              <a class="navbar-item navbar-previous-title" href="{% link openseeds/index.md %}#cohorts"> Cohorts </a>
+              <a class="navbar-item navbar-previous-title" href="{% link library.md %}"> Video Library </a>
+            </div>
+          </div>
+
+          <div class="navbar-item has-dropdown is-hoverable">
+            <a class="navbar-link" href="{% link openseeds/ols-8/index.md %}">OLS-8</a>
+            <div class="navbar-dropdown">
+              <a class="navbar-item" href="{% link openseeds/ols-8/index.md %}"> Syllabus </a>
+              <a class="navbar-item" href="{% link openseeds/ols-8/schedule.md %}"> Schedule</a>
+              <a class="navbar-item" href="{% link openseeds/ols-8/projects-participants.md %}"> Projects & Participants</a>
+              <a class="navbar-item" href="{% link openseeds/ols-8/index.md %}#mentors"> Mentors & Experts </a>
+            </div>
+          </div>
+
+          <div class="navbar-item has-dropdown is-hoverable">
+            <a class="navbar-link" href="{% link code-of-conduct.md %}">Policies</a>
+            <div class="navbar-dropdown">
+              <a class="navbar-item" href="{{ site.github.owner_url }}/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md"> Code of Conduct </a>
+            </div>
+          </div>
+
+          <div class="navbar-item has-dropdown is-hoverable">
+            <a class="navbar-link" href="{% link about.md %}">About</a>
+            <div class="navbar-dropdown">
+              <a class="navbar-item" href="{% link index.md %}">OLS</a>
+              <a class="navbar-item" href="{% link people.md %}"> People </a>
+              <a class="navbar-item" href="{% link community.md %}"> Team </a>
+              <a class="navbar-item" href="{% link funders.md %}"> Funding & Supporters </a>
+              <a class="navbar-item" href="{% link partners.md %}"> Partners </a>
+              <!--<a class="navbar-item" href="/events"> Events </a>-->
+            </div>
+          </div>
+
+          <div class="navbar-item has-dropdown is-hoverable">
```

Comment:
This menu is not needed in the Open Seeds

Reply:
💯 It has been removed. Services are now only in the `default-header`.